ansible_rulebook/cli.py

```diff
@@ -258,7 +262,8 @@ def get_version() -> str:
         f"  Drools_jpy version = {importlib.metadata.version('drools_jpy')}",
         f"  Java home = {java_home}",
         f"  Java version = {java_version}",
-        f"  Python version = {''.join(sys.version.splitlines())}",
+        f"  Python version = {''.join(sys.version.splitlines())}\n"
```

Comment:
Please, can you omit this line, we are going to revisit this section anyway. As for now this is the expected output,